### PR TITLE
Fix datetime compare

### PR DESCRIPTION
**Pull request by ntoll**
## What

*sigh* Yet another bug fix. Datetimes are hard. Originally the code was trying to get the `utc` version of the date, but the wrong method was used to achieve this. Now falls back to using `Date` instances for comparison (where timezones don't play a role).

## How to review

1. Describe the steps required to test the changes
2. ???
3. Profit!

### CODE REVIEW

**Comment by ntoll** [comment]
Supersedes #838

File: controlpanel/frontend/jinja2/user-list.html [hunk 1]
```diff
@@ -29,20 +29,17 @@ <h1 class="govuk-heading-xl">{{ page_title }}</h1>
...
-          (<strong><a href="https://github.com/orgs/moj-analytical-services/people/{{ user.username }}" target="_blank">unused?</a></strong>)
-          {%- endif -%}
-          {%- endwith -%}
+          {%- if user.last_login -%}
+              <span data-last-login="{{user.last_login}}"
+                    title="{{user.last_login.strftime("%Y/%m/%d %H:%M:%S")}}">
+                {{ timesince(user.last_login) }} ago
+              </span>
           {%- else -%}
+              <span>Never logged in.</span>
```
Comment (xoen) [inline controlpanel/frontend/jinja2/user-list.html:38]: I wonder if it's worth having the link in this branch as well or not. Although I don't imagine many users will not have a this blank.

Comment (ntoll) [reply controlpanel/frontend/jinja2/user-list.html:38]: There is a link. If they've not logged in they're marked as un-used users.

**Review by xoen — APPROVED** [review]
LGTM 👍

**Comment by ntoll** [comment]
Thanks

**Pull request merged**